src/lsp/cobol_ptree/branching_statements.ml

```diff
@@ -731,6 +736,12 @@ and pp_accept_stmt ppf = function
       pp_dual_handler pp_statement ~close:Fmt.(any "END-ACCEPT")
         ppf on_exception;
       Fmt.pf ppf "@]"
+  | AcceptFromCmdLine { item; on_exception } ->
+      Fmt.pf ppf "@[ACCEPT@;<1 2>%a@ @[FROM COMMAND-LINE"
```

Comment:
Noticing we might have boxes not closed here and above.

Reply:
Not sure where to put the closing tag though (I copied the case above 😅)

Reply:
Seems okay now.